src/CLRunner.ts

```diff
@@ -433,7 +439,8 @@ export class CLRunner {
                 const currentTicks = new Date().getTime();
                 let lastActive = await state.BotState.GetLastActive()
                 let passedTicks = currentTicks - lastActive;
-                if (passedTicks > this.modelOptions.sessionTimout!) {
+                if (passedTicks > this.modelOptions.sessionTimout) {
+                    await this.CallSessionTimeoutCallback(state, [])
```

Comment:
This is currently the problem. We need to pass current list of entities to create the memory manager.

I saw in the EndSessionCallback there was extra request to the server to query the entities from the model, but that gets the entities from latest package which may be out of sync with the running bot.

We could do this something similar to this but it seems like it sets up bad practice of the SDK calling the server. Given this is already the timeout and we don't care about latency as much, wanted to make sure it's the only option.

https://github.com/microsoft/ConversationLearner-SDK/blob/master/src/CLRunner.ts#L374